framework/test/jest/integration/utils/pg-helper/index.js

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable no-console */
+const pgpLib = require('pg-promise');
+const childProcess = require('child_process');
+const { createStorageComponent } = require('./storage');
+
+this.pgpOptions = {
+	capSQL: true,
+	promiseLib: Promise,
+	noLocking: false,
+	noWarnings: true,
+};
+const pgp = pgpLib(this.pgpOptions);
+
+class PgHelper {
+	constructor({ dbName = 'lisk_dev' }) {
+		this.dbName = dbName;
+		this.cnStr = `postgres://localhost:5432/${dbName}`;
+		if (dbName.indexOf('postgres://') === 0) {
+			this.cnStr = dbName;
+		}
+
+		this.pgp = pgp(this.cnStr);
+		this.storage = null;
+	}
+
+	_dropDB() {
+		return new Promise(resolve => {
+			console.log('Dropping database instance:', this.dbName);
+			childProcess.exec(`dropdb ${this.dbName}`, () => resolve());
+		});
+	}
+
+	_createDB() {
+		return new Promise((resolve, reject) => {
+			console.log('Creating database instance:', this.dbName);
+			childProcess.exec(`createdb ${this.dbName}`, error => {
+				if (error) {
+					return reject(error);
+				}
+				return resolve();
+			});
+		});
+	}
+
+	async bootstrap() {
+		await this._dropDB();
+		await this._createDB();
+		return this.pgp.connect();
+	}
+
+	async cleanup() {
+		await this.storage.adapter.db.$pool.end();
+		await this.pgp.$pool.end();
+	}
+
+	async createStorage(options = {}, logger) {
+		const storageOptions = {
+			database: this.dbName,
+			user: 'lisk',
+			password: 'password',
+			min: 10,
+			max: process.env.LISK_TEST_DB_MAX_CONNECTIONS || 2,
+			logFileName: `logs/devnet/lisk_${this.dbName}.log`,
+			noWarnings: true,
+			poolIdleTimeout: 30000,
+			reapIntervalMillis: 1000,
+			logEvents: ['error'],
+			...options,
+		};
+		this.storage = await createStorageComponent(storageOptions, logger);
+
+		return this.storage;
+	}
+
+	static async createAccount(db, account) {
```

Comment:
Why static?

Reply:
I had to pass db object I couldn't clean up the function inside PgHelper class. Also I'm not sure where to store these helper functions so I left it open to discussion.

Reply:
Still not sure if this is a good idea, I mean starting using helpers a lot. Given that these are integration tests, for me it would make sense to create the account using the existing storage methods. I get the point of having a helper for isolating potential bugs in the existing storage methods but I wonder how big can potentially an utilities folder became as we add more tests. 
I think tests should be extremely straight forward if we add many helpers we end up adding complexity and more effort is required as one needs not only to analyse the code but also get familiar with all the helper functions in the test suite. Maybe this was already settle but if not maybe we should consider alternatives?